cms/djangoapps/contentstore/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models for contentstore
+"""
+
+from django.db.models.fields import TextField
+
+from config_models.models import ConfigurationModel
+
+
+class VideoEncodingDownloadConfig(ConfigurationModel):
```

Comment:
How will this be shown to the user? I'm not clear if this will be displayed in Studio, or if is through the django admin table.